android/app/src/main/java/com/inaturalistreactnative/MainApplication.kt

```diff
@@ -35,10 +33,6 @@ class MainApplication : Application(), ReactApplication {
 
   override fun onCreate() {
     super.onCreate()
-    SoLoader.init(this, OpenSourceMergedSoMapping)
-    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
-      // If you opted-in for the New Architecture, we load the native entry point for this app.
-      load()
-    }
```

Comment:
Is this now handled by `loadReactNative`?

Reply:
Yes.